gf-ui-lib/src/components/CheckboxField/CheckboxField.tsx

```diff
@@ -0,0 +1,58 @@
+import {SmallTitle} from "../SmallTitle/SmallTitle";
```

Comment:
Check paths everywhere. We use absolute path with "src/..."

Reply:
If you don't get hints with an absolute path, you can try to set up VS Code like this:
![Снимок экрана 2023-05-03 175205](https://user-images.githubusercontent.com/103531486/235922024-5b032083-98bf-4f8c-b4a2-c7e33997b26f.png)
And choose non-relative instead of shortest.
You can use this setting for this project only

Reply:
move to #128

Reply:
If u have some style related issue, that not mean we should not try to achieve target store in other issue. Don't create more style mistake.  It's pretty easy to fix it here and don't make more work in future

Reply:
Absolute paths are not configured for this folder. Tried to use react-app-rewired (we used it in webapp and admin folders) but nothing happened.